lib/unifex/code_generator/tie_header.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Unifex.CodeGenerator.TieHeader do
+  @moduledoc """
+  Generates connective header file that includes proper header based
+  on selected interface.
+  """
+
+  import Unifex.CodeGenerator.Utils, only: [sigil_g: 2]
+
+  def generate_tie_header(name, interfaces) do
```

Comment:
- since it's already in `TieHeader` module, `generate` name would be sufficient
- add specs